string zone
int16 controlId
string name
int16 numVal
peac_bridge/Device device